pub-critic/src/app/register/page.tsx

```diff
@@ -0,0 +1,16 @@
+import { Backdrop } from "components/backdrop/Backdrop";
+import classes from "./page.module.scss";
+import tempBackground from "assets/gaming.webp";
+import LoginForm from "components/forms/loginForm";
+import RegisterForm from "components/forms/registerForm";
+
+const Login = () => {
+  return (
+    <div className={classes.container}>
+      <Backdrop image={tempBackground} />
```

Comment:
This can be a `ModalDialog` component that accepts childen - in this case `<RegisterForm />`